### PR TITLE
refactor: update links to module specific docs & homepage links in pubspecs

### PR DESCRIPTION
**Pull request by russellwheatley**
## Description

See [FLUTTERFIRE-34](https://yt.invertase.io/youtrack/issue/FLUTTERFIRE-34)
## Related Issues

None
## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: website/plugins.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ module.exports = [
     name: 'Analytics',
     pub: 'firebase_analytics',
     firebase: 'analytics',
+    documentation:'https://firebase.flutter.dev/docs/analytics/overview',
```
Comment (Ehesp) [inline website/plugins.js:17]: We can probably make these relative links.

File: website/src/pages/index.tsx [hunk 2]
```diff
@@ -118,9 +120,17 @@ function Home() {
                       <img width={25} src={useBaseUrl('img/firebase-logo.png')} alt="Firebase" />
                     </a>
                   </td>
+                  <td>
+                    {plugin.documentation.length ? (
+                      <a href={plugin.documentation} target="_blank">
```
Comment (Ehesp) [inline website/src/pages/index.tsx:125]: Can we use the Link component to make this relative rather than opening in a new tab?

**Review by Ehesp — CHANGES_REQUESTED** [review]

File: docs/overview.mdx [hunk 3]
```diff
@@ -255,6 +255,10 @@ get started:
...
+| **Cloud Storage**   | Cloud Storage is designed to help you quickly and easily store and serve user-generated content, such as photos and videos. <br /><br /> [View documentation &raquo;](storage/overview.mdx)                                                                                                                                             |   [firebase_storage](https://pub.dev/packages/firebase_storage)      |
+| **Crashlytics**     | Crashlytics helps you to collect analytics and details about crashes and errors that occur in your app. <br /><br /> [View documentation &raquo;](crashlytics/overview.mdx)                                                                                                                                                             |   [crashlytics](https://pub.dev/packages/crashlytics)                |
```
Comment (Salakar) [inline docs/overview.mdx:262]:
```suggestion
| **Crashlytics**     | Crashlytics helps you to collect analytics and details about crashes and errors that occur in your app. <br /><br /> [View documentation &raquo;](crashlytics/overview.mdx)                                                                                                                                                             |   [firebase_crashlytics](https://pub.dev/packages/firebase_crashlytics)                |
```

File: website/src/pages/index.tsx (hunk 2 above)
Comment (Salakar) [reply website/src/pages/index.tsx:125]: Lets do this in another PR.

File: website/plugins.js (hunk 1 above)
Comment (Salakar) [reply website/plugins.js:17]: Lets do this in another PR.

**Review by Salakar — APPROVED** [review]

**Pull request merged**